newIndex.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=
+    , initial-scale=1.0">
+
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="newStyles.css">
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <title>Nordic Rose</title>
+</head>
+
+<body>
+    <artcile>
+        <header class="header">
+
+            <div class="top_padding_nordic">
+                <h1>
```

Comment:
you have multiple `h1` tags on your page. normally a page should have only one `h1` tag